application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/Macro.xml

```diff
@@ -278,7 +278,7 @@ if (typeof(XWiki) == "undefined" || typeof(XWiki.widgets) == "undefined" || type
       // call constructor from ModalPopup with params content, shortcuts, options
       $super(
         // this element will end up as 'this.content'
-        new Element('form', {'class' : 'xform'}),
+        new Element('form', {'class' : 'xform', 'id' : 'edit-form'}),
```

Comment:
This ID is a predefined value from somewhere? Asking because it looks pretty generic

Reply:
Is an ID that I added. I changed it to something more specific.